network-area-diagram/src/main/java/com/powsybl/nad/layout/LayoutParameters.java

```diff
@@ -14,8 +14,9 @@
 public class LayoutParameters {
     private boolean textNodesForceLayout = false;
     private double springRepulsionFactorForceLayout = 0.0;
-    private Point textNodeFixedShift = new Point(100, -15);
+    private Point textNodeFixedShift = new Point(100, -40);
     private int maxSteps = 1000;
+    private double detailedTextNodeYShift = 25;
```

Comment:
you should rename it to make the edge connection explicit, something like `textNodeEdgeConnectionYShift`

Reply:
done